src/SamplesApp/UITests.Shared/Helpers/VisualTreeHelperEx.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+
+namespace UITests.Helpers
+{
+	/// <summary>
+	/// Contains various methods to debug the visual tree
+	/// </summary>
+	internal static class VisualTreeHelperEx
```

Comment:
Why do we need one more version of "GetParent" / "GetChild" / "GetDebugIdentifier" / "ShowLocalVisualTree"? We already have way to much versions / variants of this kind of helpers. We should not add another set while we are still trying to remove the deprecated ones ... unless those helpers are coming from MUX?